e2e/README.md

```diff
@@ -0,0 +1,113 @@
+# E2E Tests
```

Comment:
Please see my notes on formatting the README on the dispensing PR. That all applies here too.

Reply:
thanks @ibacher ,, ready for another review